content-src/experiments/color.yaml

```diff
@@ -1,7 +1,7 @@
 id: 17
 title: 'Color'
 slug: color
-is_featured: true
+is_featured: false
```

Comment:
We may not want to unfeature this yet.
I did make a note about adding `dev: true` to the lockbox.yaml below though, so feel free to ignore this comment temporarily until you read that. At which point you're welcome to ignore this comment permanantly.

Reply:
>We may not want to unfeature this yet.

Good point, it was good to flip for testing but needs to remain in place before merging. 👍